src/licensedcode/data/licenses/ccl.yml

```diff
@@ -0,0 +1,5 @@
+key: ccl
```

Comment:
what about using a `cockroach` key instead? No offense meant to cockroaches, of course :)